### PR TITLE
External Database Connector RNs going into 10.12 RNs

### PR DESCRIPTION
**Pull request by SurabhiSaraf**
Added release notes for 10.12 release of External database connector.

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/10/10.12.md [hunk 1]
```diff
@@ -95,6 +95,15 @@ For more information on how to set up CSP, see [Content Security Policy](/howto/
 - We made several improvements to the microflow editor. When dragging microflow/nanoflow calls into a flow (including from the **App Explorer**), we now correctly set the parameters of the call. Setting the parameters of the call now also takes the names of the variables into account and will match them by name to the parameter when possible. Thank you Peter Mudde for this suggestion! {TW}
 
 - We aligned the pagination controls position of Data Grid 2 to the default bottom position while generating overview pages or converting from data grid to data grid 2.
+- We have below addition to External Database Connector 
+- We added the following additions to External Database Connector: 
+    - GA for Snowflake connection
+    - Support for connecting to Snowflake using Key-Pair Authentication
+    - Passing NULL values to parameters
+    - Connection detail constants are now saved in active Aonfiguration of user, and Password and Private Key are now stored as private values
```
Comment (quinntracy) [inline content/en/docs/releasenotes/studio-pro/10/10.12.md]: Can you clarify the bullet "Connection detail constants are now saved in Active Configuration of user"? What is considered the "active configuration"? (I will fix the existing typo after when I update this bullet.)

**Comment by ConnorLand** [comment]
@MariaShaposhnikova I can put these bullets into the RN tomorrow. Before I did, however, I wanted to check how you feel about the amount of sub-bullets.

**Comment by MariaShaposhnikova** [comment]
@ConnorLand , looks good to me!

**Comment by ConnorLand** [comment]
I will add these RNs manually, as there is a conflict in this PR. Then I'll ping Quinn to close it.

**Pull request closed without merging**